resources/views/layouts/main_layout.blade.php

```diff
@@ -18,7 +18,7 @@
         <header class="primary-header container group">
 
             <h3 class="logo">
-                <a href="/projectfinder">sQuire</a>
+                <a href="projectfinder">sQuire</a>
```

Comment:
this was updated in master, needs to be changed back to having the slash `projectfinder`->`/projectfinder`